### ISSUE TITLE
Switch to 64bit Windows chromedriver

### ISSUE THREAD

**Issue by timovd** [fhoeben/hsac-fitnesse-fixtures#525]
In the changelog of 5.2.41 "Switch to 64bit chromedriver" is mentioned.
Linux, Mac (legacy) and M1 were already using a 64 bit driver. The Windows Chrome driver is still using the 32 bit version.

Should this line be removed from the changelog?
Or do we need to switch to 64 bit Chrome driver for Windows as well and mention the OS explicitly in the changelog?

**Comment by timovd** [comment]
@lennyg1regarding PR #521, did the 64 bit version not work?

**Comment by lennyg1** [comment]
I intended to include a 64bit chromedriver in the pull request, but during the switch to 115 I noticed a few problems with different drivers. One of the issues I ran into is that for version 115.0.5790.110 the 64bit driver was late to the party, hence I had to switch back temporarily to the 32bit version. However I forgot to revert that.

I've made a new PR #526 with the actual 64bit version for Windows too.
